assets/out

```diff
@@ -92,6 +98,13 @@ push_tags() {
   git push --tags push-target $forceflag
 }
 
+add_and_push_notes() {
+  if [ -n "$notes" ]; then
+    git notes add -m "${notes}"
```

Comment:
Hm, I think I was thrown by the fact I'd need to reference the file outside of the repo. i.e.

`git notes add -F "../${notes_file}"`

What do you think?